tools/functional-tester/etcd-tester/tester.go

```diff
@@ -129,13 +139,25 @@ func (tt *tester) runLoop() {
 				}
 				continue
 			}
-			log.Printf("etcd-tester: [round#%d case#%d] all members are consistent!", i, j)
-			log.Printf("etcd-tester: [round#%d case#%d] succeed!", i, j)
+			log.Printf("etcd-tester: [round#%d case#%d] all members are consistent with storage hashes", i, j)
+
+			revToCompact := currentRevision / 2
```

Comment:
this is a bad algo. as the revision grows, we will leave more and more uncomplicated revisions. (100 will leave 50; 1 million will leave half million). 

We should give this a const. like revToCompact = max(0, currentRevision - 10000)